packages/contracts-bedrock/test/kontrol/README.md

```diff
@@ -148,7 +142,7 @@ function setUp() public {
     superchainConfig = SuperchainConfig(superchainConfigProxyAddress);
 }
 ```
-Note that the names of the addresses come from [`DeploymentSummary.t.sol`](deployment/DeploymentSummary.t.sol) and are automatically generated by the [`make-summary-deployment.sh`](./scripts/make-summary-deployment.sh) script.
+Note that the names of the addresses automatically generated by the [`make-summary-deployment.sh`](./scripts/make-summary-deployment.sh) script.
```

Comment:
nit: 
```suggestion
Note that the names of the addresses is automatically generated by the [`make-summary-deployment.sh`](./scripts/make-summary-deployment.sh) script.
```

Reply:
using 'are' is better than 'is' here? @AmadiMichael

Reply:
Yes, true